src/main/java/com/github/cameltooling/lsp/internal/codeactions/ConvertCamelKPropertyFileModelineRefactorAction.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cameltooling.lsp.internal.codeactions;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.CodeActionKind;
+import org.eclipse.lsp4j.CodeActionParams;
+import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.TextDocumentItem;
+import org.eclipse.lsp4j.TextEdit;
+import org.eclipse.lsp4j.WorkspaceEdit;
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+
+import com.github.cameltooling.lsp.internal.CamelTextDocumentService;
+import com.github.cameltooling.lsp.internal.completion.modeline.CamelKModelineOptionNames;
+import com.github.cameltooling.lsp.internal.modelinemodel.CamelKModeline;
+import com.github.cameltooling.lsp.internal.modelinemodel.CamelKModelineOption;
+import com.github.cameltooling.lsp.internal.modelinemodel.ICamelKModelineOptionValue;
+import com.github.cameltooling.lsp.internal.parser.CamelKModelineParser;
+import com.github.cameltooling.lsp.internal.parser.ParserFileHelperUtil;
+
+public class ConvertCamelKPropertyFileModelineRefactorAction {
+	
+	public static final String CODE_ACTION_TITLE_CONVERT_PROPERTY_FILE = "Convert deprecated modeline options property-file to property=file: notation";
```

Comment:
I'm balanced if I need to put more details here, like between which and which version it is useful but I felt it was creating a too long and not enough readable string in UI.
The quick action appears anyway only when there is the deprecated notation used.

Reply:
looks good to me